### ISSUE TITLE
Reproducing results

### ISSUE THREAD

**Issue by pawankmrs** [intersun/PKD-for-BERT-Model-Compression#9]
Nice paper! Thanks for sharing the code. 
I was trying to reproduce your results. It would be great if you could share the best hyperparameter for each GLUE task?
For example with the command: `$ python NLI_KD_training.py`

For RTE I was able to get following results: 
```
acc = 0.6216666666666667
eval_loss = 1.3263624415118644
```

BUT

With only change at line 34 to `argv = get_predefine_argv('glue', 'MRPC', 'finetune_student')` 
I got:
```
acc = 0.28289855072463765
acc_and_f1 = 0.14144927536231883
eval_loss = 3.820818234373022
f1 = 0.0
```

**Comment by davidlowjw** [comment]
I'm also facing same the issue of trying to replicate the result from the paper. 
Appreciate if @intersun could help out on this :)

**Comment by intersun** [comment]
Actually I believe all hyper-parameters are recorded at 
PKD-for-BERT-Model-Compression/result/glue/result_summary/

For example, the hyper-parameters for PKD is in pkd_6layer_all.csv for each task, and the meaning of file/parameter could be inferred from their names

kd.cls.True_MRPC_nlayer.6_lr.2e-05_T.20.0_alpha.0.7_beta.8.0_bs.32-run-2

cls.True/False: stands if we normalized cls token
nlayer.6  : number of layers = 6
lr.2e-5    : learning rate is 2e-5
...
and etc. 

Please reply if you have any further questions.

Regarding your MRPC results, it looks like something is wrong (teacher's prediction and etc), if you try vanilla KD, does it work?

**Comment by pawankmrs** [comment]
Thanks! All parameter given in PKD-for-BERT-Model-Compression/result/glue/result_summary/ are certainly helpful. That clears my doubt. Thanks a lot.

I was able to figure out that when 'Debug=True', results were off. (Line 34 in NLI_KD_training.py)

Although some values are even higher than in the paper, as:
kd.cls.False_QQP_nlayer.6_lr.5e-05_T.20.0_alpha.0.7_beta.100.0_bs.32-run-1 | 4.0 | 0.9082859262923572 | 0.6082037955145293
